lib/cli/commands/listorderhistory.ts

```diff
@@ -0,0 +1,19 @@
+import { callback, loadXudClient } from '../command';
+import { Arguments } from 'yargs';
+import { ListOrderHistoryRequest } from '../../proto/xudrpc_pb';
+
+export const command = 'listorderhistory <order_id>';
+
+export const describe = 'list all events related to the order since entered until fully removed from the orderbook';
```

Comment:
This is a little long for a command line description, how about just "lists all events related to an order"